src/components/filter/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import Button from "components/base/button";
+
+function FilterBtnContainer() {
```

Comment:
Can we rename the folder by matching our main function here? Like from `filter` to `filterBtnContainer`?

One more thing from my side here is that getting the components folder separated is great in case of file/ function responsibility. But in this case on our todo-app, we only need the component FilterBtnContainer only once, right? In that case can't we keep this directly in the components folder? Cause we are making the folder structure messy when we could have a simple one. 🙂